.zeplin/components.json

```diff
@@ -0,0 +1,132 @@
+{
+  "plugins": [
+    {
+      "name": "@zeplin/cli-connect-react-plugin",
+      "config": {
+        "tsDocgen": "react-docgen-typescript"
+      }
+    },
+    {
+      "name": "@zeplin/cli-connect-storybook-plugin",
+      "config": {
+        "url": "https://future-standard.github.io/scorer-ui-kit/storybook/",
+        "fetchStories": false,
```

Comment:
Set to false to provide the path for Storybook manually since sync didn't worked automatically

Reply:
👍 I'll take a look at naming on the design / Zeplin side to see if we can get this better matching so it can be used in the future.